src/Microsoft.Azure.WebJobs/ODataFilterResolutionPolicy.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.WebJobs.Host.Bindings
+{
+    /// <summary>
+    /// Marker type for Table resolution
+    /// </summary>
+    internal sealed class ODataFilterResolutionPolicy
```

Comment:
This is the "marker" class. I've actually added "Marker" to the end of it to make that clear. I can make the "real" one public, though.